src/main/java/frc/robot/lib/interfaces/Arm.java

```diff
@@ -75,31 +79,43 @@ public Arm() {
         RobotMap.leftElbowMotor.configReverseSoftLimitThreshold(Constants.ELBOW_MOTOR.B_TRAVEL_LIMIT);
         RobotMap.rightElbowMotor.configReverseSoftLimitThreshold(Constants.ELBOW_MOTOR.B_TRAVEL_LIMIT);
 
+        // TODO: config mount pose of IMU
+
         RobotMap.rightShoulderMotor.set(ControlMode.Follower, Constants.CAN_IDS.LEFT_SHOULDER_MOTOR);
         RobotMap.rightElbowMotor.set(ControlMode.Follower, Constants.CAN_IDS.LEFT_ELBOW_MOTOR);
     }
 
     public void setIdle() {
+        // TODO: IMU degree
         moveArmPosition(100.0, 0.0);
     }
 
     public void setElbowIdle() {
         RobotMap.leftShoulderMotor.getSelectedSensorPosition();
-        RobotMap.leftElbowMotor.set(TalonFXControlMode.Position, Constants.ELBOW_IDLE.ELBOW_POSITION);
+
+        // TODO: figure out if reading pitch or roll
+        elbowCommand = controller.calculate(RobotMap.elbowIMU.getRoll(), Constants.ELBOW_IDLE.ELBOW_POSITION);
+        RobotMap.leftElbowMotor.set(elbowCommand);
+        
         elbowTarget = Constants.ELBOW_IDLE.ELBOW_POSITION;
     }
 
+    // encoder position for the shoulder, IMU degree for the elbow
     public void moveArmPosition(double shoulder, double elbow) {
         RobotMap.leftShoulderMotor.set(TalonFXControlMode.Position, shoulder);
-        RobotMap.leftElbowMotor.set(TalonFXControlMode.Position, elbow);
+
+        // TODO: figure out if reading pitch or roll
+        elbowCommand = controller.calculate(RobotMap.elbowIMU.getRoll(), elbow);
```

Comment:
This is not going to work. Previously we were sending down a target to the motor PID and the Motor PID was calculating the motor output every loop. Now we need to do this calculation loop ourselves. This function should only set the target of the PID, and then the PID calculation needs to occur every loop. Best way I can see of doing this is to make a periodic function in this class and call it in the overall robot periodic function. Also I am not sure how follower mode works, but we might need to set the power for both elbow motors instead of just one.